### ISSUE TITLE
ModuleNotFoundError: No module named 'model_filemanager'  ?? Yesterday was fine

### ISSUE THREAD

**Issue by ewebgh33** [comfyanonymous/ComfyUI#4976]
### Expected Behavior

Started ComfyUI. Immediate error, as in the title of the post and below.
12hrs ago it was fine.
I have not updated, I have not changed anything else on the computer. Not even a windows update. Was using it last night, then today it wont start.
Rebooted just to make sure
Still error on start
I don't get it, I haven't changed ANYTHING.

### Actual Behavior

In terminal/cmd, it began to start up, then stopped with the error:

```
 File "C:\AI\Image\ComfyUI_windows_portable\ComfyUI\server.py", line 30, in <module>
    from model_filemanager import download_model, DownloadModelStatus
ModuleNotFoundError: No module named 'model_filemanager'
C:\AI\Image\ComfyUI_windows_portable>pause

```
This is basically as soon as it begins, at this point:
Total VRAM 24563 MB, total RAM 65459 MB
pytorch version: 2.4.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
Using pytorch cross attention

Then  error

### Steps to Reproduce

Start from `run_nvidia_gpu.bat` as I always do, as was working 12hrs ago.

### Debug Logs

```powershell
Total VRAM 24563 MB, total RAM 65459 MB
pytorch version: 2.4.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
Using pytorch cross attention
Traceback (most recent call last):
  File "C:\AI\Image\ComfyUI_windows_portable\ComfyUI\main.py", line 91, in <module>
    import server
  File "C:\AI\Image\ComfyUI_windows_portable\ComfyUI\server.py", line 30, in <module>
    from model_filemanager import download_model, DownloadModelStatus
ModuleNotFoundError: No module named 'model_filemanager'
```

### Other

This seems to happen a lot. NO updates from me, but randomly it stops working.

**Comment by ltdrdata** [comment]
Your ComfyUI installation  is broken.

Goto ComfyUI dir in cmd.
And execute 
```
git status
```

What is the message?

**Comment by ewebgh33** [comment]
See that's the thing, last night it was working, I closed everything and shut down the computer. Next day, boot up, enter windows, double click shortcut icon... something is messed up? Why? How?
This has happened before.

I updated (git pull) and the error went away and it started.

But the thing I don't get is, literally doing nothing, taking zero actions, not even a Windows update, but in between turning the machine off at night then on in the morning, it's... borked?

Next time I will do a git status and see what it says. But this is the 2nd or 3rd time I've booted up one morning just to see an error that was not present the night before. I didn't even install any new nodes, NOTHING new. So weird.
